preset/overrides.scss

```diff
@@ -1,5 +1,12 @@
 @import './variables.scss';
 
+@media (max-width: 588px) {
```

Comment:
💭 Are there breakpoint variables that we can use?

Reply:
Sweet nice call, I've got this working another way now, including these commits:

https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark/pull/10/commits/0934e1423a0ba0c87112bd261d7af94005ec33b2

https://github.com/storypark/storyjar/pull/4495/commits/3878845c1573f1cc9ce13e71df42fd4d4e2a4e31